### ISSUE TITLE
Group and Matrix PDF generation: Show prices without Foodcoop Marge

### ISSUE THREAD

**Issue by Schaussi** [foodcoops/foodsoft#445]
Hi again,
we have now changed our Foodcoop Marge from 0% to 1%, which works fine basically.
The problem in our case is now: Our supplier gets the "Group" or "Matrix" PDF as order, because he needs to know the order group names (to write them onto the packages).
Unfortunately, the price column on those 2 PDFs includes the Foodcoop marge, which confuses the supplier.
Because we need the order group names on the PDF, we cannot use the "Fax PDF" template.
Is there a simple way to modify the price column?
Thank you

**Comment by wvengen** [comment]
Hi, thanks for your question.
That's interesting: the "Group" or "Matrix" PDF is meant for the foodcoop, while the "Fax PDF"/"Supplier PDF" is meant for the supplier. Does the supplier already package into boxes for each group? (That would be convenient!)

There is no simple way to modify the price column (except setting the margin to 0, as you start with). A solution would need to be developed, either by creating a plugin that overrides this (perhaps we'd need to add an additional method to the PDF generators to obtain the price), or a configurable option selecting the price to show in the PDFs. Perhaps both prices should be shown (because the group/matrix PDFs may still be used by the members to see what they've received and how much it costs).

**Comment by Schaussi** [comment]
Exactly, the supplier writes the order group names onto the packages.
Unfortunately, I am not very familiar with Ruby so for now, I have modified my local order_by_groups.rb and order_matrix.rb files to show "price" instead of "fc_price". I know this is not a good solution but it works for now.

**Comment by wvengen** [comment]
Sounds like a reasonable workaround for the moment.
I've opened PR #448 so that a plugin could be created that overrides `OrderMatrix.article_price`. Instead of `article.price.fc_price`, you could return `article.price.price` or `article.price.gross_price`.

Writing a plugin is explained in [plugin development](https://github.com/foodcoops/foodsoft/wiki/Plugin-development), and one would override a method in a way similar to e.g. [here](https://github.com/foodcoops/foodsoft/blob/master/plugins/messages/lib/foodsoft_messages/user_link.rb). For anyone who would be interested to pick it up.
